pkg/Microsoft.NETCore.Platforms/runtime.json

```diff
@@ -234,6 +234,13 @@
             "#import": [ "rhel", "linux-x64" ]
         },
 
+        "rhel.6": {
```

Comment:
Thanks for the info @tmds I think we will keep it as @janvorli has it then until we have any evidence to suggest otherwise.